### PR TITLE
Add a waffle flag for adding course_modes to courses when courses are…

### PR DESCRIPTION
**Pull request by reillz10**
… added to curriculum

### CODE REVIEW

**Review by schenedx — APPROVED** [review]
👍

**Comment by reillz10** [comment]
travis run quality

File: course_discovery/apps/course_metadata/tests/test_waffle.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+
+from course_discovery.apps.course_metadata.waffle import masters_course_mode_enabled
+from waffle.testutils import override_switch
+
```
Comment (iloveagent57) [inline course_discovery/apps/course_metadata/tests/test_waffle.py:5]: I think you need another newline here for the quality check to pass.

File: course_discovery/apps/course_metadata/tests/test_waffle.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+
+from course_discovery.apps.course_metadata.waffle import masters_course_mode_enabled
+from waffle.testutils import override_switch
```
Comment (iloveagent57) [inline course_discovery/apps/course_metadata/tests/test_waffle.py]: This should be grouped with the third-party libraries section of imports (i.e. under the django import above).

File: course_discovery/apps/course_metadata/waffle.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"""
+This module contains configuration settings via
+waffle switches for the course_metadata app.
+"""
+import waffle
+
+# Switches
+MASTERS_COURSE_MODE_ENABLED = u'masters_course_mode_enabled'
+
```
Comment (iloveagent57) [inline course_discovery/apps/course_metadata/waffle.py:9]: This is one-too-many newlines :)

**Review by iloveagent57 — COMMENTED** [review]
Looks good, just some minor quality cleanup.

**Pull request merged**